java/clients/src/test/java/sleeper/clients/teardown/ShutdownSystemProcessesIT.java

```diff
@@ -234,7 +316,8 @@ void shouldNotTerminateEMRClusterWhenClusterIsTerminated() throws Exception {
             stubFor(listActiveClustersRequest()
                     .willReturn(aResponse().withStatus(200).withBody("" +
                             "{\"Clusters\": []}")));
-
+            stubFor(listActiveApplicationsRequest()
+                    .willReturn(aResponseWithNumRunningApplications(0)));
             // When
```

Comment:
This is missing an empty line before the When.

Reply:
Added